iprange/iprange_test.go

```diff
@@ -10,28 +10,31 @@ import (
 func TestSimpleAddress(t *testing.T) {
 	ipRange, err := Parse("192.168.1.1")
 	assert.Nil(t, err)
+	assert.Len(t, ipRange, 1)
 
-	assert.Equal(t, net.IPv4(192, 168, 1, 1).To4(), ipRange.Min)
-	assert.Equal(t, ipRange.Min, ipRange.Max)
+	assert.Equal(t, net.IPv4(192, 168, 1, 1).To4(), ipRange[0].Min)
```

Comment:
Rather than changing ipRange into a slice, I think adding ipRangeList or something similar would be better. This `[0]` makes extra runtime checks necessary (what if there's no `[0]`?)